01week/rockPaperScissors.js

```diff
@@ -7,17 +7,60 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+// hand1 input rock,paper,scissors
+// check if input is valid (strings of rock, paper, scissors)
+// hand2 input rock,paper,scissors
+// check if input is valid
+// Check for a winner or tie ()
+
 
 function rockPaperScissors(hand1, hand2) {
+  // global variables and an array
+  const inputs = ['rock', 'paper', 'scissors'];
+
+  const Scrubhand1 =  hand1.toLowerCase();
+  const Scrubhand2 =  hand2.toLowerCase();
 
-  // Write code here
+  let x = false;
+  let y = false;
 
+// Check to see if user inputs are valid
+  for (let i=0; i < inputs.length; i++) {
+    if (Scrubhand1 === inputs[i]) {
```

Comment:
How would this check if the input is valid? Let's say you call this function with ROck. Scrubhand would be rock and for every letter in ROck, you would check to see if each letter is equal to 'rock'. This check would always fail